app/controllers/conference_controller.rb

```diff
@@ -15,6 +15,9 @@ def schedule
     @rooms = @conference.venue.rooms if @conference.venue
     @events = @conference.program.events
     @dates = @conference.start_date..@conference.end_date
+    confirmed_events = @conference.program.events.where(state: 'confirmed')
+    @scheduled_events = confirmed_events.where('start_time IS NOT NULL').order(:start_time)
```

Comment:
The scheduled scope should not check if the event is confirmed? I think so. It depends if we want to consider the unconfirmed event with start_time scheduled or not and consequently showing them in the schedule. That scope is already used in the schedule, so I think I should use it here and maybe modified what the scope does.